src/br/com/ciandt/csartini/service/ComentarioService.java

```diff
@@ -0,0 +1,57 @@
+package br.com.ciandt.csartini.service;
+
+/**
+* Método que realiza a remoção do comentário
+* Exemplo: {@code Boolean seraExcluidoSe = comentarioRemovido(comentario)
+* @param comentario {@link} variavel
+* @return se verdadeiro ou falso
+* 
+* @author csartini
+**/
+
+/**
+ * metodo construtor acima
+ * 
+ * @author csartini
+ *
+ */
+public class ComentarioService {
+
+	/**
+	 * Método que realiza a remoção do comentário Exemplo:
+	 * {@code Boolean seraExcluidoSe = comentarioRemovido(comentario) @param
+	 * comentario {@link} variavel
+	 * 
+	 * @return se verdadeiro ou falso
+	 * 
+	 * @author csartini
+	 **/
+	private final static String COMENTARIO_REMOVIDO = new String("Comentario removido!");
```

Comment:
Geralmente não usamos javadoc para variáveis
```suggestion
        //Variável Constante com mensagem de comentário removido
	private final static String COMENTARIO_REMOVIDO = new String("Comentario removido!");
```